component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -707,7 +732,9 @@ private void updateUserAttribute(String username, Map<String, String> attribute,
             verifyUserExists(username, tenantDomain);
             UserStoreManager userStoreManager = userRealm.getUserStoreManager();
             userStoreManager.setUserClaimValues(username, attribute, null);
-        } catch (UserStoreException | AuthenticationFailedException e) {
+        } catch (UserStoreException e) {
+            throw e;
```

Comment:
no need to catch this exception if we are only throwing it. we can remove this catch block

Reply:
Fixed this isssue